### PR TITLE
[@tizentv/webide-common-tizentv] fix incorrect types for samsung cert manager

### PR DESCRIPTION
**Pull request by bartvandenende-wm**
Please fill in this template.

- [x] Use a meaningful title for the pull request. Include the name of the package modified.
- [x] Test the change in your own code. (Compile and run.)
- [x] [Add or edit tests](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#my-package-teststs) to reflect the change.
- [x] Follow the advice from the [readme](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#make-a-pull-request).
- [x] Avoid [common mistakes](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#common-mistakes).
- [x] [Run `npm test <package to test>`](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/master/README.md#running-tests).

Select one of these and delete the others:

If changing an existing definition:
- [x] Provide a URL to documentation or source code which provides context for the suggested changes: https://github.com/Samsung/webIDE-common-tizentv/blob/master/lib/samsungCertificateManager.js
https://github.com/Samsung/webIDE-common-tizentv/blob/master/lib/profileManager.js
https://github.com/Samsung/webIDE-common-tizentv/blob/master/lib/profileEditor.js

- [ ] If this PR brings the type definitions up to date with a new version of the JS library, update the version number in the header.

### CODE REVIEW

**Review by victor-wm — APPROVED** [review]
Changes LGTM

File: types/tizentv__webide-common-tizentv/index.d.ts [hunk 1]
```diff
@@ -14,9 +14,19 @@ export interface AuthorInfo {
   departmentInfo: string;
   emailInfo: string;
 }
+export interface SamsungAuthorInfo {
```
Comment (victor-wm) [inline types/tizentv__webide-common-tizentv/index.d.ts:17]: nit: whitespace before

**Review by victor-wm — APPROVED** [review]
Looks good!

**Comment by bartvandenende-wm** [comment]
Ready to merge

**Pull request merged**